test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -33,6 +33,14 @@ parseHttpConnectionManagerFromV2Yaml(const std::string& yaml) {
   return http_connection_manager;
 }
 
+envoy::config::filter::network::http_connection_manager::v2::HttpConnectionManager
+parseHttpConnectionManagerFromV2YamlAndValidate(const std::string& yaml) {
```

Comment:
Sorry I meant for you to just change the previous `parseHttpConnectionManagerFromV2Yaml` to always validate vs. add a new function. Can you do that?

Reply:
No, I can't, or it should be done in another PR, some other case will fail by validating

Reply:
OK no problem, let's do this in a follow up please. Can you add a TODO?